### PR TITLE
Docs: updating Framework.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: updating imports in Framework. Closes cksource/ckeditor5-commercial#6168

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Review by gorzelinski — COMMENTED** [review]
I added some suggestions.

File: docs/framework/index.md [hunk 1]
```diff
@@ -28,42 +28,12 @@ The CKEditor&nbsp;5 predefined builds {@link getting-started/legacy-getting-star
...
-To start using the framework refer to:
-
-* The {@link framework/quick-start Quick start} guide.
-* The {@link framework/architecture/intro Introduction to the framework architecture} guide.
-* The {@link tutorials/crash-course/editor step-by-step crash course}.
+* Tutorials &ndash; A crash course, a basic tutorial and advanced guides for any developers keep on writing their own CKEditor&nbsp;5 plugins.
+* Development tools &ndash; A set of useful tooling and resources for anyone wanting to develop their editor.
+* Architecture &ndash; Intro-duction to the main pillars of CKEditor&nbsp;5's architecture.
+* Deep dive &ndash; Further Framework information for advanced developers.
+* How-tos &ndash; If you just need to know how to set the height of CKEditor&nbsp;5 or learn how to focus the editor, this is the right section!
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/framework/index.md:39]: In commit.

File: docs/tutorials/abbreviation-plugin-tutorial/abbreviation-plugin-level-1.md
[the remark this answers is not in the document]
Comment (godai78) [reply docs/tutorials/abbreviation-plugin-tutorial/abbreviation-plugin-level-1.md:1]: Reverted in commit.

File: docs/framework/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (godai78) [reply docs/framework/index.md:39]: This is intentional, these section have not landing pages, so it's strange to link Deep dive to Clipboard and Development tools to Inspectior difrectly, and so on.

File: docs/framework/contributing/code-style.md
[the remark this answers is not in the document]
Comment (godai78) [reply docs/framework/contributing/code-style.md:1]: In commit.

**Pull request merged**